control/src/robocup/modules/MotionControlModule.cpp

```diff
@@ -33,22 +33,24 @@ MotionControlModule::MotionControlModule(LockedStruct<BatteryVoltage>& batteryVo
 }
 
 void MotionControlModule::entry() {
-    auto motionCommandLock = motionCommand.lock();
-    auto motorCommandLock = motorCommand.lock();
-    auto motorFeedbackLock = motorFeedback.lock();
-    auto imuDataLock = imuData.lock();
-    auto batteryVoltageLock = batteryVoltage.lock();
-
-    DebugFrame frame;
-    frame.ticks = xTaskGetTickCount();
-    frame.gyro_z = imuDataLock->omegas[2];
-    frame.accel_x = imuDataLock->accelerations[0];
-    frame.accel_y = imuDataLock->accelerations[1];
+    auto motion_command = motionCommand.lock().value();
```

Comment:
Changes in this file do not completely fill in the new debug frame format. They were mostly intended to fix the fact that we were holding waaay too many locks throughout this whole function.

Later work will involve actually populating the new data (and providing logging facilities :smile:  )